client/src/main/java/io/pravega/client/stream/ScalingPolicy.java

```diff
@@ -24,7 +24,7 @@
 public class ScalingPolicy implements Serializable {
     private static final long serialVersionUID = 1L;
 
-    public enum Type {
+    public enum RetentionType {
```

Comment:
Could you share the thought process behind this change of `ScalingPolicy.Type ` to  `ScalingPolicy.RetentionType`?

Reply:
I think this is a typo. It should be ScalingType